packages/fluxible-router/package.json

```diff
@@ -2,13 +2,16 @@
   "name": "fluxible-router",
   "version": "2.0.0-beta.12.0",
   "description": "Routing for Fluxible applications",
-  "main": "index.js",
+  "main": "./dist/lib/index.js",
```

Comment:
I think you can just remove the dist folder and just having everything off of "lib" since its all under "src" now.

Reply:
Oh, my fault, this lib is not the same as the previous lib. I'll rename it to cjs to make it clear.

Reply:
Done.